fastlane/Fastfile

```diff
@@ -651,15 +653,20 @@ REPOSITORY_NAME = 'WordPress-Android'
   # If a locale doesn't have any screenshots, it'll be skipped.
   # -----------------------------------------------------------------------------------
   # Usage:
-  # bundle exec fastlane upload_and_replace_screenshots_in_play_store
+  # bundle exec fastlane upload_and_replace_screenshots_in_play_store app:<wordpress|jetpack>
   #
   # Example:
-  # bundle exec fastlane upload_and_replace_screenshots_in_play_store
+  # bundle exec fastlane upload_and_replace_screenshots_in_play_store app:wordpress
   #####################################################################################
   desc 'Upload Screenshots to Play Store and Replaces the existing ones'
   lane :upload_and_replace_screenshots_in_play_store do |options|
+    app = get_app_name_option!(options)
+    package_name = APP_SPECIFIC_VALUES[app.to_sym][:package_name]
+    metadata_dir = File.join('fastlane', APP_SPECIFIC_VALUES[app.to_sym][:metadata_dir], 'android')
+
     upload_to_play_store(
-      package_name: 'org.wordpress.android',
+      package_name: package_name,
+      metadata_path: metadata_dir,
```

Comment:
This change makes sure we will be able to use that lane for Jetpack in the future as well as WordPress.

Even if during this sprint this lane was only used with `app:wordpress` since only the WordPress screenshots were updated (ref: pbArwn-46A-p2), making this lane be compatible with Jetpack will be useful in the future.